README.md

```diff
@@ -43,3 +43,5 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 ## Support
 
 Azure Key Vault Provider for Secrets Store CSI Driver is an open source project that is [**not** covered by the Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). [Please search open issues here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues), and if your issue isn't already represented please [open a new one](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/new/choose). The project maintainers will respond to the best of their abilities.
+
+However, [Arc extension](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/tutorial-akv-secrets-provider) and [AKS addon](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-driver) for _Azure Key Vault Provider for Secrets Store CSI Driver_ are managed offerings and supported by Microsoft Azure support policy.
```

Comment:
Maybe I missed it but I could not find a link to the AKS addon or Arc extension in the website or readme. Can we update the landing page so it is very easy to discover them?

Reply:
This is the main Readme. Please let me know if there is a better way to surface this here.

I have also added it to the main landing page of the website. https://deploy-preview-1094--secrets-store-csi-driver-provider-azure.netlify.app/#managed-add-ons